ksqldb-functional-tests/src/test/resources/historical_plans/greatest_-_test_all_null_input_-_JSON/7.0.0_1623819053014/plan.json

```diff
@@ -0,0 +1,214 @@
+{
```

Comment:
This set of historic QTT plans is a duplicate of the ones below. Can you go through, check for, and remove any duplicates?